ui/src/app/views/workspace-nav-bar/component.spec.tsx

```diff
@@ -0,0 +1,68 @@
+import {currentWorkspaceStore, NavStore} from 'app/utils/navigation';
+import {WorkspaceNavBarReact} from 'app/views/workspace-nav-bar/component';
+import {mount} from 'enzyme';
+import {WorkspaceAccessLevel} from 'generated';
+import * as React from 'react';
+import {WorkspacesServiceStub} from 'testing/stubs/workspace-service-stub';
+
+describe('WorkspaceNavBarComponent', () => {
+
+  let props: {};
+  const workspace = {
+    ...WorkspacesServiceStub.stubWorkspace(),
+    accessLevel: WorkspaceAccessLevel.OWNER,
+  };
+
+  const component = () => {
+    return mount(<WorkspaceNavBarReact {...props}/>, {attachTo: document.getElementById('root')});
+  };
+
+  beforeEach(() => {
+    props = {};
+
+    currentWorkspaceStore.next(workspace);
+  });
+
+  it('should render', () => {
+    const wrapper = component();
+    expect(wrapper).toBeTruthy();
+  });
+
+  it('should highlight the active tab', () => {
+    props = {tabPath: ''};
+    const wrapper = component();
+    expect(wrapper.find({'data-test-id': 'About', 'aria-selected': true}).exists()).toBeTruthy();
+  });
+
+  it('should navigate on tab click', () => {
+    const navSpy = jest.fn();
+    NavStore.navigate = navSpy;
+    const wrapper = component();
+
+    wrapper.find({'data-test-id': 'Cohorts'}).first().simulate('click');
+    expect(navSpy).toHaveBeenCalledWith(
+      ['/workspaces', workspace.namespace, workspace.id, 'cohorts']);
+  });
+
+  it('should call delete method when clicked', () => {
```

Comment:
testing if the delete method gets called seems to be a test of the delete modal or the dropdown menu.... should this component just test that the right options show up in the dropdown menu?

Reply:
I want to make sure that both methods that are passed in as props are called correctly; I'm not worried that the right options won't be in the menu anymore than I am that the right tabs won't be in the bar.